src/Lambda/Input/InvocationRequest.php

```diff
@@ -107,19 +107,12 @@ public function getQualifier(): ?string
 
     public function requestBody(): string
     {
-        $document = new \DOMDocument('1.0', 'UTF-8');
-        $document->formatOutput = true;
-
-        if (null !== $input = $this->Payload) {
-            $document->appendChild($document->createElement('Payload', $input));
```

Comment:
sure?

Reply:
Nope, not yet =)

Reply:
This is wrong. 
https://docs.aws.amazon.com/lambda/latest/dg/API_Invoke.html#API_Invoke_RequestBody

I'll update

Reply:
Fixed. =)